tools/dumpobj.ml

```diff
@@ -89,7 +89,7 @@ let rec print_struct_const = function
   | Const_base(Const_string (s, _, _)) -> printf "%S" s
   | Const_immstring s -> printf "%S" s
   | Const_base(Const_char c) -> printf "%C" c
-  | Const_base(Const_int32 i) -> printf "%ldl" i
+  | Const_base(Const_int32 (_, s)) -> print_string s
```

Comment:
This is probably wrong -- I suspect that the `l` suffix should not be swallowed. There appears not to be a test for this. I'll investigate.